contracts/eosio.system/native.hpp

```diff
@@ -0,0 +1,83 @@
+/**
+ *  @file
+ *  @copyright defined in eos/LICENSE.txt
+ */
+#pragma once
+
+#include <eosiolib/types.hpp>
+
+namespace eosiosystem {
+   template <account_name SystemAccount>
+   class native {
+      public:
+         ACTION(SystemAccount, newaccount) {
+            EOSLIB_SERIALIZE(newaccount, BOOST_PP_SEQ_NIL)
```

Comment:
I think you forgot to add the fields for the actions.  These should be the same as the native actions.